src/test/java/com/snowflake/kafka/connector/internal/streaming/SchematizationUtilsTest.java

```diff
@@ -90,10 +91,11 @@ public void testGetColumnTypesWithSchema() throws JsonProcessingException {
             System.currentTimeMillis(),
             TimestampType.CREATE_TIME);
 
-    Map<String, String> columnToTypes =
-        SchematizationUtils.getColumnTypes(
+    Map<String, Pair<String, String>> columnInfos =
```

Comment:
You should also test a schema without the `doc` field to have complete test case coverage.

Reply:
add test case